gpt-survey-summarizer/values.yaml

```diff
@@ -21,14 +24,39 @@ redis:
   auth:
     # -- Redis password
     password: ""
+  # -- Configuration to add to redis.conf
+  commonConfiguration: |-
+    # Enable AOF https://redis.io/topics/persistence#append-only-file
+    appendonly yes
+    #
+    # Backups:
+    # * After 14400 seconds (4 hours) if at least 1 change was performed
+    # * After 3600 seconds (1 hour) if at least 100 changes were performed
+    # * After 600 seconds (10 minutes) if at least 10.000 changes were performed
+    # 
+    save 14400 1 3600 100 600 10000
+    # Workdir of redis
+    dir /data
+    # Database file name
+    dbfilename dump.rdb
+
+databaseDumpExport:
+  # -- Whether to enable exporting of the database dump
+  enabled: true
```

Comment:
**suggestion:** We could add a `suspended` value here to allow [schedule suspensions](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#schedule-suspension).

```suggestion
databaseDumpExport:
  # -- Whether to enable exporting of the database dump
  enabled: true
  # -- Suspend the automatic execution
  suspended: false
```

Compare to `enabled: false`, this would allow to still declare the `CronJob` in `suspended` mode, then we can trigger the Job manually with:
```
kubectl create job --from=cronjob/dump-exporter job-01
```